src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -11,23 +11,32 @@ export default class ContentCreatorComponent extends Component {
 
         this.handleIframeLoad = this.handleIframeLoad.bind(this);
         this.handlePublish = this.handlePublish.bind(this);
+        this.enablePublishButton = this.enablePublishButton.bind(this);
+        this.disablePublishButton = this.disablePublishButton.bind(this);
+        this.renderPublishBtn = this.renderPublishBtn.bind(this);
+
+        this._refIframe = null;
+        this._refContentContainer = null;
 
         this.state = {
             iframeLoading: true,
+            publishBtnDisabled: false,
```

Comment:
AFAIK in our code we prefer boolean variables prefixed with _is_, so here we would have `isPublishBtnDisabled`. I drew this conclusion after my PRs reviews, I might be wrong. 😛

Reply:
See the line above. I just keep the convention from that file. In new file, we should name the `boolean` props/attrs/variables with `is` prefix.